cityofaustin/atd-moped
# shows substantial completion date plus advanced search

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/19257

## Testing
**URL to test:** 

https://deploy-preview-1453--atd-moped-main.netlify.app/

**Steps to test:**
- go to projects list view, 'Substantial completion date' should be visible by default right before 'Start date'
- sort the projects by that field, try ascending, descending and unsorted
- export the table and confirm the column is visible and populated in the csv
- add an advanced search filter for the new field and try different operators and values

---
#### Ship list
